docs/index.md

```diff
@@ -0,0 +1,35 @@
+# Welcome to MkDocs
+
+For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+
+## Commands
+
+* `mkdocs new [dir-name]` - Create a new project.
+* `mkdocs serve` - Start the live-reloading docs server.
```

Comment:
should we add this one to the Makefile?

Reply:
They don't require the make command, so I would leave these here for now.